tools/README.md

```diff
@@ -0,0 +1,79 @@
+# prepare_lava_job
+
+prepare_lava_job is a Python script designed to generate Job files in YAML format. These job files are designed to be used with LAVA in order to automate the process of flashing and testing RevPi devices, thus saving time in identifying errors in the images.
+
+Once the LAVA Job has been generated, it can be submitted using one of the following methods:
+- lavacli command-line tool
+- LAVA webUI
+- LAVA REST/XML-RPC API
+
+Please refer to the LAVA documentation for detailed instructions on how to submit the generated job. For a example using lavacli, see the "Example" section.
+
+## Features
+
+- Generates LAVA Job files for flashing and testing images on RevPi devices.
+- Allows customization of job parameters through command-line arguments.
+
+## Usage
+
+The LAVA Job Generator can be executed using the following command:
+
+- -d, --device-type: Specify the device type to use for testing.
+- -u, --url: Provide the URL of the image to be flashed.
+- --flash: Specify the manifest used to flash the given image onto the DUT (required).
+- -s, --testsuite: Specify the job testsuite to be used after flashing (required).
+- -t, --tags: Specify the tag used for selecting an appropriate device for testing.
+
+> **_NOTE:_**  By default, the generated job is configured for a RevPi Core device (device_type: RevPi_Core) and uses the "config_core_01" tag.
+
+> **_NOTE:_**  The default image is "2023-06-26-revpi-bullseye-rc3-arm64-lite.img". 
+
+> **_NOTE:_**  The default behavior can be modified by providing appropriate options.
```

Comment:
It would be helpful to mention that these defaults depend on what "template" is given and what the respective parameters are defined in there.

Reply:
I updated the readme file...